frontend/src/Form/beta/ModelCardForm.tsx

```diff
@@ -0,0 +1,120 @@
+import { Box, Divider, Stack, StepButton, StepLabel, Stepper } from '@mui/material'
+import MaterialStep from '@mui/material/Step'
+import { useTheme } from '@mui/material/styles'
+import Form from '@rjsf/mui'
+import { DescriptionFieldProps } from '@rjsf/utils'
+import validator from '@rjsf/validator-ajv8'
+import React, { Dispatch, SetStateAction, useState } from 'react'
+import CustomTextInput from 'src/MuiForms/CustomTextInput'
+import TagSelector from 'src/MuiForms/TagSelectorBeta'
+import { setStepState } from 'utils/beta/formUtils'
+
+import { SplitSchemaNoRender } from '../../../types/interfaces'
+import Nothing from '../../MuiForms/Nothing'
+
+export default function ModelCardForm({
+  splitSchema,
+  setSplitSchema,
+  canEdit = false,
+}: {
+  splitSchema: SplitSchemaNoRender
+  setSplitSchema: Dispatch<SetStateAction<SplitSchemaNoRender>>
+  canEdit?: boolean
+}) {
+  const [activeStep, setActiveStep] = useState(0)
+  // TODO - add validation
+  const [_openValidateError, _setOpenValidateError] = useState(false)
```

Comment:
By using `_` we are saying that our intention is to not use these vars. If this is just because it's WIP, I would argue that the linting warnings are exactly what we want so `_` could be removed.